src/tests/ftest/recovery/pass_1.py

```diff
@@ -17,18 +17,104 @@ class Pass1Test(TestWithServers):
     :avocado: recursive
     """
 
+    def test_run_checker(self, inconsistency=None, policies=None):
+        """Run DAOS checker with kinds of options.
+
+        1. Enable check mode.
+        2. Run checker under dry-run mode.
+        3. Set repair policy as interaction.
+        4. Run checker under auto mode.
+        5. Reset repair policy as default.
+        6. Run checker under regular mode, that will repair the inconsistency.
+        7. Disable check mode.
+
+        Jira ID: DAOS-13047
+        """
+        errors = []
+
+        dmg_command = self.get_dmg_command()
+        # 1. Enable check mode.
+        dmg_command.check_enable()
+
+        # 2.1 Start checker with "dry-run" option.
+        # that will detect the inconsistency but not really fix it.
+        dmg_command.check_start(dry_run=True)
+
+        # 2.2 Query the checker.
+        query_msg = ""
+        for _ in range(8):
+            check_query_out = dmg_command.check_query()
+            if check_query_out["response"]["status"] == "COMPLETED":
+                query_msg = check_query_out["response"]["reports"][0]["msg"]
+                break
+            time.sleep(5)
```

Comment:
This snippet is used three times. It would be good to add a helper method for this